### ISSUE TITLE
NotImplementedError: Unknown infobox block name 'Category'

### ISSUE THREAD

**Issue by ISSOtm** [DragoonAethis/itch-dl#1]
Hi!
```
INFO:root:Downloading https://andreipasynkov.itch.io/upandown
Traceback (most recent call last):
  File "/usr/lib/python3.10/runpy.py", line 196, in _run_module_as_main
    return _run_code(code, main_globals, None,
  File "/usr/lib/python3.10/runpy.py", line 86, in _run_code
    exec(code, run_globals)
  File "/tmp/itch-dl/itch_dl/__main__.py", line 3, in <module>
    run()
  File "/tmp/itch-dl/itch_dl/cli.py", line 68, in run
    return drive_downloads(jobs, download_to, args.mirror_web, args.api_key, keys, parallel=args.parallel)
  File "/tmp/itch-dl/itch_dl/downloader.py", line 350, in drive_downloads
    results = [downloader.download(job) for job in tqdm(jobs, **tqdm_args)]
  File "/tmp/itch-dl/itch_dl/downloader.py", line 350, in <listcomp>
    results = [downloader.download(job) for job in tqdm(jobs, **tqdm_args)]
  File "/tmp/itch-dl/itch_dl/downloader.py", line 240, in download
    metadata = self.extract_metadata(game_id, url, site)
  File "/tmp/itch-dl/itch_dl/downloader.py", line 149, in extract_metadata
    infobox = parse_infobox(infobox_div)
  File "/tmp/itch-dl/itch_dl/infobox.py", line 116, in parse_infobox
    parsed_block = parse_tr(name, content_td)
  File "/tmp/itch-dl/itch_dl/infobox.py", line 100, in parse_tr
    raise NotImplementedError(f"Unknown infobox block name '{name}' - please file a new itch-dl issue.")
NotImplementedError: Unknown infobox block name 'Category' - please file a new itch-dl issue.
```
...so I did! Got this from `https://itch.io/jam/game-boy-showdown`.

**Comment by DragoonAethis** [comment]
Whoops, that's right. I've added support for this block and published 0.2.1 which is now able to download that entire jam. `pip install --upgrade itch-dl` and give it a go :)

**Comment by ISSOtm** [comment]
Thanks!
